repos/system_upgrade/el7toel8/actors/peseventsscanner/actor.py

```diff
@@ -31,6 +31,10 @@ def get_packages(event, name):
         return (event.get(name, {}) or {}).get('package', [])
 
 
+    def report_outdated_data(self):
+        self.report_error("Found outdated PES data. Consult docs to retrieve updated data.")
```

Comment:
I think this is a good place to use StopActorExecutionError exception, makes no sense to continue if we have outdated data

Reply:
Thanks, changed that